atd-vze/src/views/Crashes/Crash.js

```diff
@@ -149,19 +157,22 @@ function Crash(props) {
   };
 
   const {
-    atd_fatality_count: deathCount,
-    sus_serious_injry_cnt: seriousInjuryCount,
     latitude_primary: latitude,
     longitude_primary: longitude,
-    address_confirmed_primary: primaryAddress,
-    address_confirmed_secondary: secondaryAddress,
     cr3_stored_flag: cr3StoredFlag,
     temp_record: tempRecord,
     geocode_method: geocodeMethod,
     cr3_file_metadata: cr3FileMetadata,
     investigator_narrative_ocr: investigatorNarrative,
   } = !!data?.atd_txdot_crashes[0] ? data?.atd_txdot_crashes[0] : {};
 
+  const {
+    crash_injury_metrics: { cris_fatality_count: deathCount },
+    crash_injury_metrics: { sus_serious_injry_count: seriousInjuryCount },
+    address_primary: primaryAddress,
+    address_secondary: secondaryAddress,
+  } = crashData?.crashes_by_pk ? crashData?.crashes_by_pk : {};
```

Comment:
as we work on the crash details page more and more, this will grow and eventually we will be able to ditch the other query.

Reply:
yep—this is great!

Reply:
+1 - love the destructuring! I think that this could use some more default values for the nested keys like `vz_fatality_count` because it will break if it ever hits that empty object fallback. It probably never will because of the loading state check further up in the component, but it is something that stuck out because it has bitten me in the past.